app/views/content/train-to-be-a-teacher/_nav-cards.html.erb

```diff
@@ -0,0 +1,39 @@
+<section class="container">
+    <div class="category__nav-cards">
+        <a href="/train-to-be-a-teacher/if-you-have-a-degree" class="category__nav-card">
+            <h2>If you have a degree</h2>
+			
+            <p>
+                Find out how to get QTS if you have a degree or you’re studying for one now. There are different options you can choose.
+            </p>
+            <span class="icon"></span>
```

Comment:
@gazcarr Icon looks great - however, VoiceOver reads it out as "heavy right pointing angle quotation mark ornament - link"

Can we mark it as decorative?

Also - does it need a better class name that 'icon' @peteryates ?

Reply:
While we're prototyping I wouldn't worry too much about the implementation details, I'll reimplement this with an eye on usability after the UR